### PR TITLE
Merge branch 'master' into staging-ln

### PR DESCRIPTION
**Pull request by strmci**

### CODE REVIEW

**Comment by benma** [comment]
It's unusual to see only one commit here. I would expect all commits from master that are not in staging, not squashed.

**Comment by strmci** [comment]
> It's unusual to see only one commit here. I would expect all commits from master that are not in staging, not squashed.
thanks, it's fixed now

**Comment by thisconnect** [comment]
I merged https://github.com/BitBoxSwiss/bitbox-wallet-app/pull/2925 to master
you might want to include it in this PR.

**Comment by strmci** [comment]
> I merged #2925 to master you might want to include it in this PR.

thanks, yes, I updated it to include recent commits from master.

**Comment by thisconnect** [comment]
iirc there was some conflict in go.mod or go.sum @Beerosagos ptal

File: frontends/web/src/routes/router.tsx [hunk 1]
```diff
@@ -171,8 +172,9 @@ export const AppRouter = ({
     <LightningDeactivate />
   </InjectParams>;
 
-  const BuyInfoEl = <InjectParams>
-    <BuyInfo
+
```
Comment (Beerosagos) [inline frontends/web/src/routes/router.tsx:175]: nit: please, remove the empty line :)

**Review by Beerosagos — APPROVED** [review]
tACK, with just a small nit

File: frontends/web/src/routes/router.tsx (hunk 1 above)
Comment (strmci) [reply frontends/web/src/routes/router.tsx]: removed

**Pull request merged**